test/cypress/integration/users-management-ui.spec.js

```diff
@@ -0,0 +1,159 @@
+import {
+  TEST_ID_USER_LIST_TABLE,
+  TEST_ID_DETAIL_USER_TABLE,
+  TEST_ID_USER_SEARCH_FORM,
+  TEST_ID_USER_PROFILE_FORM,
+  TEST_ID_USER_LIST_PAGE,
+  TEST_ID_USER_FORM,
+  TEST_ID_USER_AUTHORITY_TABLE,
+  TEST_ID_USER_AUTHORITY_PAGE_FORM, TEST_ID_USER_AUTHORITY_MODAL,
+} from '../../../src/ui/test-const';
```

Comment:
did you check if it's possible to use absolute imports? it's appbuilder's codebase standard, not sure if it's possible to do that between test and src